File: Treinamento/Java/Estrutura_de_decisão/Questão02.java
import java.util.Scanner;
public class Questão02 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o número desejado: ");
        int n = s.nextInt();

        if (n >= 0) {
            System.out.println(n + " é um número positivo");
        } else {
            System.out.println(n + " é um número negativo");
        }
    }
}
        

File: Treinamento/C/Estrutura_de_decisão/Questão02.c
#include <stdio.h>

int main() {
    int n;

    printf("Digite o número desejado: ");
    scanf("%i", &n);

    if (n < 0) {
        printf("%i é um número negativo\n", n);
    } else {
        printf("%i é um número positivo\n", n);
    }
}
